Blueprints/BlueprintDefinitions/vs2017/AspNetCoreWebAPI/template/test/BlueprintBaseName.1.Tests/S3ProxyControllerTests.cs

```diff
@@ -2,6 +2,7 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
+using System.Text.Json;
```

Comment:
The blueprints in VS2017 folder target .NET Core 2.1 which doesn't have System.Text.Json. I suggest leaving the 2017 blueprints alone.

Reply:
Okay, reverted those changes now 👍